### ISSUE TITLE
Cannot create conda environment (ResolvePackageNotFound)

### ISSUE THREAD

**Issue by SaltedFishLZ** [TwentyBN/smth-smth-v2-baseline-with-models#4]
Hi, following the README, I got such an error in setting the environment:

```bash
Solving environment: failed

ResolvePackageNotFound: 
  - nbconvert==5.3.1=py_1
```

Can anyone help me? Thanks a lot.

**Comment by raghavgoyal14** [comment]
Thanks for pointing it out!
After a quick search I gathered that this has been encountered previously: https://github.com/ContinuumIO/anaconda-issues/issues/9480

I will update the environment file sometime next week and let you know.

**Comment by chen-ming2019** [comment]
@SaltedFishLZ Hi , I trid to unzip the dataset serveral times,but failed with a unkown end error .  so,I only got some part of datas  .   Can you succeed in unzipping the dataset and geting all datas?

**Comment by SaltedFishLZ** [comment]
Yes, I can. Did you follow the guide in their official sites? Something-something datasets are sliced into multiple compressed files.

**Comment by chen-ming2019** [comment]
@SaltedFishLZ ok ,I have solved the problem. Thank you.

**Comment by gyzhang123** [comment]
same problem

**Comment by raghavgoyal14** [comment]
#6

**Comment by anuragkumar** [comment]
Hi,

I am trying to create the conda environment using your environment.yml file. I am using ubuntu 20.04 distribution. It is failing because of conflict issues for packages. I am not sure how to resolve it.
May I know which platform you used for creating the environment? Is it possible to provide environment.yml file for Ubuntu?

Thanks

**Comment by raghavgoyal14** [comment]
Can you please try downloading the required packages individually? I can't test the repo on 20.04 since I have 18.04 LTS (although I'm not sure whether this is even the source of error).

In retrospect, I shouldn't have included build versions in the yaml file which becomes incompatible with newer versions of conda.
